tools/deb/description-discovery

```diff
@@ -0,0 +1 @@
+Istio Pilot Discovery provides services discovery for the http://istio.io project.
```

Comment:
add "service" maybe ? (or "server")
also same comment as in the other PR re newlines at the end of file (configure your editor?)

Reply:
Resolved by removing the 'discovery' deb for now :-) 
There are other issues to solve before running pilot discovery outside k8s - for 0.2 we only need the agent.

We can have a discussion on the mailing list about proper names, directory structure and distribution 
- maybe adding people more familiar with distributions ( since I assume we may also have distro-maintained packages )